applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_incompressible_potential_element.cpp

```diff
@@ -263,5 +257,31 @@ namespace Kratos {
         KRATOS_CHECK(EquationIdVector[i] == i);
       }
     }
+
+    // Checks the function GetPotentialOnNormalElement
+    KRATOS_TEST_CASE_IN_SUITE(GetPotentialOnNormalElement, CompressiblePotentialApplicationFastSuite)
+    {
+      Model this_model;
+      ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+      GenerateElement(model_part);
+      Element::Pointer pElement = model_part.pGetElement(1);
+
+      // Define the nodal values
+      Vector potential(3);
+      potential(0) = 0.0;
+      potential(1) = 1.0;
+      potential(2) = 2.0;
+
+      for (unsigned int i = 0; i < 3; i++)
+        pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY_POTENTIAL) = potential(i);
+
+      auto potentials = PotentialFlowUtilities::GetPotentialOnNormalElement<2,3>(*pElement);
+
+      KRATOS_CHECK_NEAR(potentials(0), 0.0, 1e-7);
```

Comment:
As pointed out by @rubenzorrilla in another PR, it's better to store the reference value in a std::vector and loop over it to reduce the number of KRATOS_CHECK_NEAR macro calls (see other cpp tests).